dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracleAppConfig.scala

```diff
@@ -104,7 +104,7 @@ case class DLCOracleAppConfig(
     val key =
       WalletStorage.getPrivateKeyFromDisk(seedPath,
                                           SegWitMainNetPriv,
-                                          password,
+                                          Some(password),
```

Comment:
same here, is there are ason the parameter to `initialize()` is not a `passwordOpt: Option[AesPassword]` ?